dev/requirements.txt

```
@@ -4,7 +4,9 @@ pip
 boltons>=23.0.0
 conda>=23.3.1
 importlib-metadata
-libmamba>=1.4.1
-libmambapy>=1.4.1
+libmamba==1.4.1
+libmambapy==1.4.1
```

Maybe we can leave it like this
```suggestion
libmamba>=1.4.1,!=1.4.2
libmambapy>=1.4.1,!=1.4.2
```

---

On 2nd thought, see https://github.com/conda/conda-libmamba-solver/pull/187#issuecomment-1509764952

That would imply reverting this change.

---

Should we apply this change? I'm worried this will immediately start failing when 1.4.3 is released.

---

I worry that if we lock the version like that, other problems arise in future releases and we don't see them.